tests/test_tab.py

```diff
@@ -0,0 +1,67 @@
+"""
+ Pylibui test suite.
+
+"""
+
+from pylibui.controls import Tab, Button
+from tests.utils import WindowTestCase
+
+
+class TabTest(WindowTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.tab = Tab()
+
+    def test_can_append_pages_to_tab(self):
+        """Tests that we can append pages to tab."""
```

Comment:
You should try to improve a little bit the description of the tests. In this case you are not testing that you can append pages to tab, but that the control correctly returns the number of pages. The assert is what you are really testing...